Classes8.1/ExceptionHandling.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace Classes8._1
+{
+    public class ExceptionHandling
+    {
+        public static void ShowMassiveElement()
+        {
+            int[] massive = { 8, 7, 1, 4, 2 };
+
+            Console.WriteLine("Input index of element in massive:");
+            try
+            {
+                string? inputedValue = Console.ReadLine();
+                string? checkedValue = inputedValue.Equals(string.Empty) ? null : inputedValue;
+                int inputtedNumber = int.Parse(checkedValue);
+                int massiveElement = massive[inputtedNumber];
+                Console.WriteLine($"Massive element that has index {inputedValue} has value {massiveElement}");
+            }
+            catch (FormatException e) when (LogException(e))
```

Comment:
Yep, it works. But I don't get the point of using "when" condition. Usually it' being used as additional condition to handle this exception and in your case it's always true

Reply:
I hope, I've fixed it 🥲